task4.py

```diff
@@ -0,0 +1,14 @@
+# Напишите программу, которая будет преобразовывать десятичное число в двоичное.
+# Пример:
+# - 45 -> 101101
+# - 3 -> 11
+# - 2 -> 10
+
+x = int(input('введите число: '))
+
+bin_x = ''
+while x>0:
+    bin_x = str(x%2)+ bin_x
+    x //=2
```

Comment:
```suggestion
while x:
    bin_x = str(x % 2) + bin_x
    x //= 2
```
Почему ж вы так не любите пробелы?